sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md

```diff
@@ -109,7 +110,31 @@ _The following was run on Windows 10 and Ubuntu Desktop 20.04 environments, with
 
 5. Create a sketch on Arduino IDE for the IoT Hub telemetry sample.
 
-    - Clone the [Azure SDK for Embedded C](https://github.com/Azure/azure-sdk-for-c) repository locally and then open the [ESP8266 sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/aziot_esp8266) (from the local clone) on the Arduino IDE.
+    - Clone the [Azure SDK for Embedded C](https://github.com/Azure/azure-sdk-for-c) repository locally and 
+
+    - Generate the `ca.h` header (in the ESP8266 sample folder!) with the public root CA for server certificate validation
+
+      - Navigate to the ESP8266 sample in your local clone
+
+        ```bash
+        cd <cloned repo root>/sdk/samples/iot/aziot_esp8266
+        ```
+
+      - Run the script to generate the `ca.h` header
+
```

Comment:
Nit: Add a '.' to the end of instruction sentences.

Reply:
Fixed.